### PR TITLE
[Feature] sc-31808 Draft of the hint for OSS user to the cloud

### PR DESCRIPTION
**Pull request by kentwelcome**
**PR checklist**

- [x] Ensure you have added or ran the appropriate tests for your PR.
- [x] DCO signed

**What type of PR is this?**
Feature

**What this PR does / why we need it**:
Hint the OSS user when they open the report page with Lineage Diff info.

***Single Report***
Clipboard 2023-02-08 at 4 50 48 PM

***Comparison Report***
CleanShot 2023-08-04 at 10 09 07

**Which issue(s) this PR fixes**:
sc-31808

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

### CODE REVIEW

File: static_report/src/components/Reports/ReportContextBar.tsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { BiPlug } from 'react-icons/bi';
 import { BsGearWideConnected } from 'react-icons/bs';
 import { GoGitBranch } from 'react-icons/go';
 import { borderVal } from '../../utils';
+import { CloudPlusIcon } from '../../utils/icons';
```
Comment (popcornylu) [inline static_report/src/components/Reports/ReportContextBar.tsx:7]: Remove this because is it not used.

File: static_report/src/utils/icons.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { IconType } from 'react-icons/lib';
```
Comment (popcornylu) [inline static_report/src/utils/icons.tsx:1]: You can put this code to `static_report/src/components/Icons/index.tsx`

**Review by popcornylu — CHANGES_REQUESTED** [review]
You can merge this two minor and merge directly.

File: static_report/src/components/Reports/ReportContextBar.tsx (hunk 1 above)
Comment (kentwelcome) [reply static_report/src/components/Reports/ReportContextBar.tsx]: removed

File: static_report/src/utils/icons.tsx (hunk 2 above)
Comment (kentwelcome) [reply static_report/src/utils/icons.tsx]: Moved

**Review by popcornylu — APPROVED** [review]

**Pull request merged**